dwitter/views_v2.py

```diff
@@ -17,10 +20,17 @@
 from dwitter.utils import length_of_code
 
 
-class UserViewSet(mixins.RetrieveModelMixin, viewsets.GenericViewSet):
+class UserViewSet(viewsets.ModelViewSet):
```

Comment:
Does this accidentally expose update, partial_update, delete etc on user?

Reply:
Good catch. Been a while since I've properly read up on DRF; this would have been a very bad thing to merge 🙃

Reply:
Added tests for this.